Library/Formula/freetds.rb

```diff
@@ -21,43 +19,52 @@ class Freetds < Formula
   end
 
   option :universal
-  option "enable-msdblib", "Enable Microsoft behavior in the DB-Library API where it diverges from Sybase's"
-  option "enable-sybase-compat", "Enable close compatibility with Sybase's ABI, at the expense of other features"
-  option "enable-odbc-wide", "Enable odbc wide, prevent unicode - MemoryError's"
-  option "enable-krb", "Enable Kerberos support"
+  option "with-msdblib",       "Enable Microsoft behavior in the DB-Library API where it diverges from Sybase's"
+  option "with-sybase-compat", "Enable close compatibility with Sybase's ABI, at the expense of other features"
+  option "with-odbc-wide",     "Enable odbc wide, prevent unicode - MemoryError's"
+  option "with-krb5",          "Enable Kerberos support"
+  option "without-openssl",    "Build without OpenSSL support (default is to use brewed OpenSSL)"
+
+  deprecated_option "enable-msdblib"       => "with-msdblib"
+  deprecated_option "enable-sybase-compat" => "with-sybase-compat"
+  deprecated_option "enable-odbc-wide"     => "with-odbc-wide"
+  deprecated_option "enable-krb"           => "with-krb5"
 
   depends_on "pkg-config" => :build
   depends_on "unixodbc" => :optional
-  depends_on "openssl"
+  depends_on "openssl" => :recommended
 
   def install
-    system "autoreconf -i" if build.head?
+    system "autoreconf", "-i" if build.head?
 
     args = %W[
       --prefix=#{prefix}
-      --with-openssl=#{Formula["openssl"].opt_prefix}
       --with-tdsver=7.1
       --mandir=#{man}
     ]
 
-    if build.with? "unixodbc"
-      args << "--with-unixodbc=#{Formula["unixodbc"].prefix}"
-    end
+    # Allow Homebrew developer mode to engage turning any warnings about
+    # configuration options into fatal errors
+    args << "--enable-option-checking=fatal" if ARGV.homebrew_developer?
 
-    if build.include? "enable-msdblib"
-      args << "--enable-msdblib"
+    # OpenSSL has historically been an absolute requirement of this formula.
+    # However, the FreeTDS docs assert that linking with OpenSSL introduces a
+    # licensing conflict.  We're splitting the difference by making it a
+    # recommended option.
```

Comment:
The historic default of the _formula_ is to have an absolute dependency on (Homebrew's) OpenSSL. But FreeTDS itself defaults to _not_ linking against OpenSSL at all unless it's requested.

I was afraid if I just yanked it, it might break something for someone already using Homebrew's freetds.